### PR TITLE
Fix IAR error when building simulator arbiter.

### PR DESCRIPTION
**Pull request by LuDuda**

### CODE REVIEW

File: src/raal/simulator/nrf_raal_simulator.c [hunk 1]
```diff
@@ -46,8 +46,6 @@
 #include "nrf.h"
 #include "nrf_raal_config.h"
 
-#include <cmsis/cmsis_gcc.h>
```
Comment (hubertmis) [inline src/raal/simulator/nrf_raal_simulator.c]: It seems that header file defining NVIC_* function is missing now. Isn't it?

Comment (LuDuda) [reply src/raal/simulator/nrf_raal_simulator.c]: nrf.h includes nrf52.h which includes proper cmsis file which is cmsis_core4.h

Comment (hubertmis) [reply src/raal/simulator/nrf_raal_simulator.c]: Is it possible to include it directly?

Comment (LuDuda) [reply src/raal/simulator/nrf_raal_simulator.c]: Isn't that idea of nrf.h to include all chip related functionality? But okay i will include it.

Comment (hubertmis) [reply src/raal/simulator/nrf_raal_simulator.c]: I'm OK to keep this include indirectly through `nrf.h` in this case.

**Pull request merged**